watershed-iOS/Watershed/Watershed/WPEditTaskViewController.m

```diff
@@ -0,0 +1,55 @@
+//
+//  WPEditTaskViewController.m
+//  Watershed
+//
+//  Created by Jordeen Chang on 4/12/15.
+//  Copyright (c) 2015 Blueprint. All rights reserved.
+//
+
+#import "WPEditTaskViewController.h"
+#import "WPNetworkingManager.h"
+
+@implementation WPEditTaskViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    [self preloadFields];
+    self.navigationItem.title = @"Edit Site";
+}
+
+// Override
+//- (void)updateServerWithSite:(WPSite *)site {
+////    site.siteId = self.site.siteId;
+//    
+//    // Don't request the list of sites, because it is already called in the ViewController's viewWillAppear
+//    __weak __typeof(self)weakSelf = self;
+//    [[WPNetworkingManager sharedManager] editTaskWithTask:
+//                                               parameters:[[NSMutableDictionary alloc] init]
+//                                                  success:^(WPTask *task) {
+//        __strong __typeof(weakSelf)strongSelf = weakSelf;
+//        [strongSelf dismissSelf];
+//    }];
+//}
```

Comment:
Should this be here? Not sure why we would need to request sites in the task editing view controller.